src/main.cpp

```diff
@@ -63,8 +63,20 @@ int main(int argc, char* argv[])
 
         std::clog <<
             "Starting xeus-sqlite kernel...\n\n"
-            "If you want to connect to this kernel from an other client, you can use"
-            " the " + file_name + " file."
+            "If you want to connect to this kernel from an other client, just copy"
+            " and paste the following content inside of a `kernel.json` file. And then run for example:\n\n"
+            "# jupyter console --existing kernel.json\n\n"
+            "kernel.json\n```\n{\n"
+            "    \"transport\": \"" + config.m_transport + "\",\n"
+            "    \"ip\": \"" + config.m_ip + "\",\n"
+            "    \"control_port\": " + config.m_control_port + ",\n"
+            "    \"shell_port\": " + config.m_shell_port + ",\n"
+            "    \"stdin_port\": " + config.m_stdin_port + ",\n"
+            "    \"iopub_port\": " + config.m_iopub_port + ",\n"
+            "    \"hb_port\": " + config.m_hb_port + ",\n"
+            "    \"signature_scheme\": \"" + config.m_signature_scheme + "\",\n"
+            "    \"key\": \"" + config.m_key + "\"\n"
+            "}\n```\n"
```

Comment:
That doesn't work, you need to pass the right file name, and remove the `kernel.json` stuffs in this case.

```suggestion
            "# jupyter console --existing " + file_name + "\n\n"
```